iosApp/iosApp/Login/LoginView.swift

```diff
@@ -70,6 +70,7 @@ struct LoginView: View {
 			.background(LinearGradient(colors: [.orange, .red], startPoint: .top, endPoint: .bottom))
 			.cornerRadius(10)
 			.padding(.bottom)
+            .accessibilityIdentifier("enterNewmButton")
```

Comment:
not sure why it's showing up like this here, but looks fine in XC

Reply:
Looks fine to me... 🤷 
![image](https://user-images.githubusercontent.com/595982/152452539-47c18083-0c2a-446f-8631-3ac329c30b82.png)